### PR TITLE
Rename SetDead

### PR DESCRIPTION
**Pull request by qndel**
Found this name in 1.08 address dump, makes much more sense :)

### CODE REVIEW

File: Source/dead.cpp [hunk 1]
```diff
@@ -76,7 +76,7 @@ void AddDead(Point tilePosition, int8_t dv, Direction ddir)
 	dDead[tilePosition.x][tilePosition.y] = (dv & 0x1F) + (ddir << 5);
 }
 
-void SetDead()
+void SyncUniqDead()
```
Comment (julealgon) [inline Source/dead.cpp:79]: Can we avoid abbreviations? Is this for unique monsters? If so, can we use "Unique"?

Comment (julealgon) [inline Source/dead.cpp:79]: Also, what does "Sync" mean here? I think we should rename this to something that makes more sense/is more specific.

Comment (qndel) [reply Source/dead.cpp:79]: dunno, I literally copy/pasted the name

Comment (AJenbo) [reply Source/dead.cpp:79]: i think its related to loading them from saves

**Review by AJenbo — APPROVED** [review]

**Pull request merged**